components/UWAPlantMap.vue

```diff
@@ -49,6 +75,19 @@ export default {
     markers() {
       // return the array
       return [...plants]
+    },
+    defaultInfo() {
+      return {
```

Comment:
probs should have a better default value

Reply:
Please elaborate :0

Reply:
The names here, pretty weird having plant name as planticus namium lol

Reply:
Oh yeah sure np